.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing Interoperability
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: workflow_dispatch
+jobs:
+  Testing_Interoperability:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Downloads assets
+        uses: robinraju/release-downloader@v1.7
+        with:
+          latest: true
+          fileName: "*"
+      - name: Unzip
+        uses: montudor/action-zip@v1
+        with:
+          args: unzip -qq *.zip -d executables
+      - name: Run Interoperability script
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -r requirements.txt
+          cd executables
+          for i in ./* ; do for j in ./*; do python3 ./../interoperability_report.py -P $i -S $j -o=./../general_report.xml; done; done
```

Comment:
can we put this in several lines? (for a better readability)

Reply:
okey